developer/graphql-intro.md

```diff
@@ -0,0 +1,150 @@
+
+## Your first query
+
+To try a simple query, open the GraphiQL interface by going to `http://localhost:3000/graphql` in your browser.
+
+In the right-hand query box, enter the following:
+
+```js
+{
+  primaryShopId
+}
+```
+
+You should see a successful response:
+
+```js
+{
+  "data": {
+    "primaryShopId": "cmVhY3Rpb24vc2hvcDpKOEJocTN1VHRkZ3daeDNyeg=="
+  }
+}
+```
+
+Running this query through the GraphiQL interface is the equivalent of running this [GET query](http://localhost:3000/graphql-alpha?query=%7B%0A%20%20primaryShopId%0A%7D) to `http://localhost:3000/graphql-alpha` with this query string (`query=%7B%0A%20%20primaryShopId%0A%7D`) in the browser.
+
+The GraphiQL interface allows you to not only make queries, but also read the corresponding documentation and go through your history.
+
+## Your first authenticated query
+
+While some queries, such as `primaryShopId`, do not require being logged in, other queries do require authentication and may either error or return different results depending on your authentication and authorization status.
+
+For example, let's fetch the `viewer`'s name, without logging in:
+
+```js
+{
+  viewer {
+    name
+  }
+}
+```
+
+You will not get any data:
+
+```js
+{
+  "data": {
+    "viewer": {
+      "name": null
+    }
+  }
+}
+```
+
+To authenticate, simply log in to a shop as any user in a browser tab in the same browser.
+
+- Open `http://localhost:3000`
+- Log in as an user
+
+Click the Play button to replay the `viewer` query again:
+
+```js
+{
+  "data": {
+    "viewer": {
+      "name": "Reaction User"
+    }
+  }
+}
+```
+
+To also get the viewer's email addresses, run:
+
+```js
+{
+  viewer {
+  	name
+    emailRecords {
+      provides
+      address
+      verified
+    }
+	}
+}
+```
+
+You should see the email address you used to log in to this user's account included in the result.
+
+## A query with a parameter
+
+First, find your viewer account's ID by using this query:
+
+```js
+{
+   viewer {
+      _id
+   }
+}
+```
+
+Then, copy the `id` and pass it in as a parameter in the next query for an `account`:
+
+```js
+{
+   account(id: "cmVhY3Rpb24vYWNjb3VudDpaN3pTcVlHZzJNbU1wazlTRw==") {
+     name
+   }
+}
+```
+
+## Adding and updating data with mutations
+
+Now, we'll try making a `mutation` to update a user's currency code with the `setAccountProfileCurrency` method.
+
+Pass in the `accountId` and `currencyCode`  as inputs to the mutation:
+
+```js
+mutation {
+  setAccountProfileCurrency(input: {
+    accountId: "cmVhY3Rpb24vYWNjb3VudDpaN3pTcVlHZzJNbU1wazlTRw==",
+    currencyCode: "EUR"
+  })
+}
+```
+
+To confirm that the mutation changed the currency as expected, you can query the viewer's currency code with:
+
+```js
+{
+  viewer {
+    currency {
+      code
+    }
+  }
+}
+```
+
+Once you've added an entry successfully, check in your database or your running application to confirm the changes were made correctly.
```

Comment:
I don't think the phrase "added an entry" makes sense here anymore now that the example is changing the currency.